mods/persistence/modules/chargen/vending/industrial.dm

```diff
@@ -12,11 +12,15 @@
 		/obj/item/chargen_box/industrial/engineering = 999,
 		/obj/item/chargen_box/industrial/firefighter = 999,
 		/obj/item/chargen_box/industrial/building = 999,
-		/obj/item/chargen_box/industrial/microlathe = 999,
+		/obj/item/chargen_box/industrial/steel = 999,
```

Comment:
How necessary are the steel and glass refill boxes now that we have the public refinery, and the autolathe includes some steel?

Reply:
Mostly useful for building structures as they are quite resource-intensive. As the 'building material kit' already includes steel and glass, I figured having specific packs for different materials would reduce clutter if someone was just trying to get steel instead of the entire kit.